/**
 * 从堆栈中一个函数的获取调用者的信息
 * @param caller 如果支持`Error.captureStackTrace`，则使用caller定位
 * @param deep 否则直接使用手动计数定位
 */
export declare const captureErrorStackTrace: (target: Object, caller: Function) => Object;
export declare const GetCallerInfo: (caller: Function) => string;
export declare class CustomError extends Error {
    get name(): string;
    static call(_this: any, message?: string): CustomError;
}
